test_runner/batch_others/test_normal_work.py

```diff
@@ -0,0 +1,38 @@
+from fixtures.log_helper import log
+from fixtures.zenith_fixtures import ZenithEnv, ZenithEnvBuilder, ZenithPageserverHttpClient
+
+
+def check_tenant(env: ZenithEnv, pageserver_http: ZenithPageserverHttpClient):
+    tenant_id, timeline_id = env.zenith_cli.create_tenant()
+    pg = env.postgres.create_start('main', tenant_id=tenant_id)
+    res = pg.safe_psql_many(queries=[
+        'CREATE TABLE t(key int primary key, value text)',
+        'INSERT INTO t SELECT generate_series(1,100000), \'payload\'',
+        'SELECT sum(key) FROM t',
+    ])
+
+    assert res[-1][0] == (5000050000, )
+    # TODO check detach on live instance
+    log.info("stopping compute")
+    pg.stop()
+    log.info("compute stopped")
+
+    pg.start()
+    res = pg.safe_psql('SELECT sum(key) FROM t')
+    assert res[0] == (5000050000, )
+
+    pageserver_http.timeline_detach(tenant_id, timeline_id)
+
+
+def test_normal_work(zenith_env_builder: ZenithEnvBuilder):
+    """
+    Basic test, write something in setup with wal acceptors,
+    ensure that successfully commits.
+    Check several timelines.
```

Comment:
NIT: Might worth adding that a new tenant and timeline creation & detach happens here.